tendenci/themes/t7-base/templates/reports/membership_quick_list.html

```diff
@@ -10,14 +10,28 @@ <h1>{% trans "Current Members Quick List" %}</h1>
     {% trans 'Download CSV File' %}</a></p>
     <table class="table table-tendenci-reports">
         <tr>
-            <th>{% trans "Last Name" %}</th>
-            <th>{% trans "First Name" %}</th>
-            <th>{% trans "Company" %}</th>
+            <th><a href="{% order_by request order_by 'user__last_name' %}">{% trans "Last Name" %}</a></th>
```

Comment:
Where is this `order_by` template tag coming from?

Reply:
Oops, forgot to stage and commit the tag. Fixed.